src/framework/components/camera/component.js

```diff
@@ -121,6 +124,10 @@ const properties = [
  * both front and back faces will be rendered. Defaults to true.
  * @property {boolean} flipFaces If true the camera will invert front and back faces. Can be useful
  * for reflection rendering. Defaults to false.
+ * @property {number} aperture The aperture of the camera lens in f-stops. High values means small aperture,
+ * low values means high aperture. Default is 1.0.
+ * @property {number} shutter The shutter speed of the camera in seconds. Default is 1.
```

Comment:
Is 1s default so lighting looks correct?

Since the common case for rendering on monitors is 60hz, I would have expected a default shutter speed of 1/60?

Reply:
Yeah, having it at 1s, f/1 and 100 ISO looks like it produces the same exposure as before. We can tweak them later, but for the current lighting parameters, these settings are good.